al-jamiah-journal-of-islamic-studies.csl

```diff
@@ -0,0 +1,497 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" initialize="false" initialize-with="." page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Al-Jami'ah - Journal of Islamic Studies</title>
+    <title-short>AJIS</title-short>
+    <id>http://www.zotero.org/styles/al-jamiah-journal-of-islamic-studies</id>
+    <link href="http://www.zotero.org/styles/al-jamiah-journal-of-islamic-studies" rel="self"/>
+    <link href="http://aljamiah.or.id/index.php/AJIS/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Al-Jamiah</name>
+      <email>editor@aljamiah.org</email>
+      <uri>www.aljamiah.org</uri>
+    </author>
+    <category citation-format="note"/>
+    <issn>0126-012X</issn>
+    <eissn>2338-557X</eissn>
+    <summary>note with bibliography (adapted-modified from several sources)</summary>
```

Comment:
> `<summary>note with bibliography (adapted-modified from several sources)</summary>`

@aljamiah, is there a CSL style you used as the main base for this style? If so, we'd like to add a "template" link to that style (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link).

Reply:
Thanks!